### PR TITLE
Remove references to contacting or emailing Steve

### PR DESCRIPTION
**Pull request by aesharpe**
### Describe the Policy Change
Not a policy change, just removing references to contact Steve.

### Reasoning and Concerns
Steve is no longer with the coop, and we don't want to encourage people to contact him about random stuff like PTO. This PR simply removes and replaces references to Steve throughout the docs. I wasn't 100% who to replace him with. Sometimes it was Zane (for tax and form stuff), sometimes it was Ben (for PTO stuff), and sometimes I just wrote the "admin team" or "us" which feels like a cop-out. If you know who to put instead, lmk!

### CODE REVIEW

File: docs/policies.md [hunk 1]
```diff
@@ -140,8 +140,8 @@ Each member will be reimbursed **up to $3,000 in any rolling 3-year period** to
 
 **Provided to:** Members only
 
-As a remote organization, Catalysters rarely see one another in person. To encourage folks to bridge the virtual divide, each member may request reimbursement for up to **$200 per year** for travel and food expenses related to visiting another member of the coop. These expenses must be _marginal and additional_ to existing plans where the primary purpose of the expenses is visiting another co-oper . Receipts must be submitted through the expense reimbursement form for approval by Steve and reimbursement.
+As a remote organization, Catalysters rarely see one another in person. To encourage folks to bridge the virtual divide, each member may request reimbursement for up to **$200 per year** for travel and food expenses related to visiting another member of the coop. These expenses must be _marginal and additional_ to existing plans where the primary purpose of the expenses is visiting another co-oper . Receipts must be submitted through the expense reimbursement form for approval and reimbursement.
```
Comment (bendnorman) [inline docs/policies.md:143]: We could add a link to the [reimbursement form](https://docs.google.com/forms/d/e/1FAIpQLSf4r4H-bqHsR_QVt_lALdBDBH_PRi96W8kPo0eCK1r-5SkZBg/viewform).

File: docs/catalyst_101.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Once the existing members have voted to offer you membership, it’s up to you t
 #### Accepting Membership
 
 - Sign the [Membership Agreement](https://drive.google.com/file/d/1tt4IXVwKi2Gj-L755AUZyu1QaVDRO8px/view?usp=sharing)
-- Purchase $1000 share, your initial capital contribution to the co-op, by sending Steve a check in the mail.
+- Purchase $1000 share, your initial capital contribution to the co-op, by sending us a check in the mail.
```
Comment (bendnorman) [inline docs/catalyst_101.md:125]: Who do we make the check out to, and what address are we using now?

**Review by bendnorman — CHANGES_REQUESTED** [review]
Looks good! Just a couple of questions.

File: docs/policies.md (hunk 1 above)
Comment (aesharpe) [reply docs/policies.md:143]: Good idea! I'll do that

File: docs/catalyst_101.md (hunk 2 above)
Comment (aesharpe) [reply docs/catalyst_101.md:125]: I'm not sure...I tagged CG in slack about this

Comment (aesharpe) [reply docs/catalyst_101.md:125]: But I agree we should include information here rather than just saying ~send us a check~

Comment (bendnorman) [reply docs/catalyst_101.md:125]: Sounds good. I'll approve once @cmgosnell gets back to you.

Comment (aesharpe) [reply docs/catalyst_101.md:125]: Ok CG says to send it to her, I'll update this

Comment (zaneselvans) [reply docs/catalyst_101.md:125]: Does it have to be a check? If someone can do an ACH transfer would we not want to do that instead?

Comment (aesharpe) [reply docs/catalyst_101.md:125]: I can text Steve about this

Comment (aesharpe) [reply docs/catalyst_101.md]: Apparently we can use ACH but there is a fee of about $10-15

**Review by bendnorman — APPROVED** [review]
Looks good! Should we just have people ask Christina for her address because these docs are public?

**Pull request merged**